frontend/src/StoryUpload/StoryUpload.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+const StoryUpload = () => {
+    const [selectedFile, setSelectedFile] = useState(null);
+
+    const handleFileChange = (event) => {
+        setSelectedFile(event.target.files[0]);
+    };
+
+    const handleUpload = async () => {
+        if (!selectedFile) return;
+
+        const formData = new FormData();
+        formData.append('story', selectedFile);
+        formData.append('userId', 1); // Assume user ID is 1 for this example
```

Comment:
For instances like this where we add some temporary logic its good practice to add a #TODO with a quick detail on the change that needs to be made. So for example here that could look like:

#TODO: Pass real userId to story upload and replace dummy data